handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -469,6 +469,49 @@ private int readPlaintextData(final ByteBuffer dst) {
         return sslRead;
     }
 
+    /**
+     * Visible only for testing!
+     */
+    final int maxWrapOverhead() {
+        return maxWrapOverhead;
+    }
+
+    /**
+     * Visible only for testing!
+     */
+    final int maxEncryptedPacketLength() {
+        return maxWrapOverhead + MAX_PLAINTEXT_LENGTH;
+    }
+
+    final int calculateMaxLengthForWrap(int plaintextLength, int numComponents) {
+        return (int) min(maxWrapBufferSize, plaintextLength + (long) maxWrapOverhead * numComponents);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Inconsistent synchronization of io.netty.handler.ssl.ReferenceCountedOpenSslEngine.maxWrapOverhead; locked 66% of time [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=findbugs%3AIS2_INCONSISTENT_SYNC)